### PR TITLE
Ready for release 0.1.0

### PR DESCRIPTION
**Pull request by PaulineMauryL**

### CODE REVIEW

File: docs/versions.yaml [hunk 1]
```diff
@@ -8,5 +8,9 @@
     - "en"
 "v0.0.4":
```
Comment (damienbfs) [inline docs/versions.yaml:9]: Should we leave this one or remove it and remove the tag?

**Comment by PaulineMauryL** [comment]
## Coverage report


 

_This PR does not seem to contain any modification to coverable code._

File: server/deploy/helm/charts/lomas_server/Chart.yaml [hunk 2]
```diff
@@ -4,9 +4,9 @@ description: Lomas deployment chart
 
 type: application
 
-version: 0.0.6
+version: 0.1.0
 
-appVersion: "0.0.4"
+appVersion: "0.1.0"
```
Comment (damienbfs) [inline server/deploy/helm/charts/lomas_server/Chart.yaml:9]: There's also the client chart to update.

**Review by damienbfs — APPROVED** [review]

File: docs/versions.yaml (hunk 1 above)
Comment (PaulineMauryL) [reply docs/versions.yaml]: i removed it

**Review by damienbfs — APPROVED** [review]

**Pull request merged**